src/components/Layout/index.jsx

```diff
@@ -86,12 +88,14 @@ class Layout extends Component {
       noSubMenu,
       pageTitle,
       isCenterPage,
+      centerWidth,
       hasBack
     } = this.props;
 
     const { isNormal, isDev, isAdmin } = this.props.user;
     const hasMenu = (isDev || isAdmin) && !isHome;
     const hasSubNav = hasMenu && !noSubMenu && !hasBack;
+    const maxWidth = isCenterPage || hasBack ? `${centerWidth}px` : '100%';
```

Comment:
refactor `maxWidth` to className is better

Reply:
There is for set center page max width， similar Dialog component props width.